colossus-metrics/src/main/scala/colossus/metrics/ConfigHelper.scala

```diff
@@ -38,6 +39,12 @@ object ConfigHelpers {
 
     def getScalaDuration(path: String): Duration = Duration(config.getString(path))
 
+    def getAddresses(path:String): Seq[String] = try {
+      config.getStringList(path).asScala
+    } catch {
+      case _: Missing => Seq("0.0.0.0")
```

Comment:
I would hold off hardcoding the string, also, please use Try rather than try.  Here we should do seq.empty and then on empty only initialize the port.  0.0.0.0 is an ipv4 specific address